src/cli.ts

```diff
@@ -30,6 +31,16 @@ const config = explorer.search();
 // If a dotenv file (or exported env vars) are provided, these override any config values
 dotenv.config();
 
+export type GlobalOptions = {
+  privateKey: string;
+  chain: helpers.ChainName;
+  providerUrl: string;
+  baseUrl: string;
+  verbose: boolean;
+  ensProviderUrl?: string;
```

Comment:
could we just merge these into a single option, so if ensProviderUrl is provided, then ens is enabled?

Reply:
I'm leaving it separate on purpose so that someone has to explicitly enable ENS experiment. I want it to be 100% clear it's an experiment, not just another option.

Reply:
ok I buy it